platform/android/java/src/org/godotengine/godot/Godot.java

```diff
@@ -748,117 +736,123 @@ public void forceQuit() {
+
+				if (s.length() == 1) {
+					s = "0" + s;
+				}
+				hexString.append(s);
+			}
+			String md5str = hexString.toString();
+
+			//Log.d("GODOT","**PACK** - My MD5: "+hexString+" - APK md5: "+main_pack_md5);
+			if (!md5str.equals(main_pack_md5)) {
+				Log.d("GODOT", "**PACK MD5 MISMATCH???** - MD5 Found: " + md5str + " " + Integer.toString(md5str.length())
+						+ " - MD5 Expected: " + main_pack_md5 + " " + Integer.toString(main_pack_md5.length()));
+				return true;
+			}
+			return false;
+		} catch (Exception e) {
+			e.printStackTrace();
+			Log.d("GODOT", "**PACK FAIL**");
+			return true;
+		}
 	}
 
 	//@Override public boolean dispatchTouchEvent (MotionEvent event) {
 	public boolean gotTouchEvent(MotionEvent event) {
 
 		super.onTouchEvent(event);
-		int evcount=event.getPointerCount();
-		if (evcount==0)
+		int evcount = event.getPointerCount();
+		if (evcount == 0)
 			return true;
 
-		int[] arr = new int[event.getPointerCount()*3];
+		int[] arr = new int[event.getPointerCount() * 3];
 
-		for(int i=0;i<event.getPointerCount();i++) {
+		for (int i = 0; i < event.getPointerCount(); i++) {
 
-			arr[i*3+0]=(int)event.getPointerId(i);
-			arr[i*3+1]=(int)event.getX(i);
-			arr[i*3+2]=(int)event.getY(i);
+			arr[i * 3 + 0] = (int) event.getPointerId(i);
+			arr[i * 3 + 1] = (int) event.getX(i);
+			arr[i * 3 + 2] = (int) event.getY(i);
 		}
 
 		//System.out.printf("gaction: %d\n",event.getAction());
-		switch(event.getAction()&MotionEvent.ACTION_MASK) {
-
-			case MotionEvent.ACTION_DOWN: {
-				GodotLib.touch(0,0,evcount,arr);
-				//System.out.printf("action down at: %f,%f\n", event.getX(),event.getY());
-			} break;
-			case MotionEvent.ACTION_MOVE: {
-				GodotLib.touch(1,0,evcount,arr);
-				//for(int i=0;i<event.getPointerCount();i++) {
-				//	System.out.printf("%d - moved to: %f,%f\n",i, event.getX(i),event.getY(i));
-				//}
-			} break;
-			case MotionEvent.ACTION_POINTER_UP: {
-				final int indexPointUp = event.getActionIndex();
-				final int pointer_idx = event.getPointerId(indexPointUp); 
-				GodotLib.touch(4,pointer_idx,evcount,arr);
-				//System.out.printf("%d - s.up at: %f,%f\n",pointer_idx, event.getX(pointer_idx),event.getY(pointer_idx));
-			} break;
-			case MotionEvent.ACTION_POINTER_DOWN: {
-				int pointer_idx = event.getActionIndex();
-				GodotLib.touch(3,pointer_idx,evcount,arr);
-				//System.out.printf("%d - s.down at: %f,%f\n",pointer_idx, event.getX(pointer_idx),event.getY(pointer_idx));
-			} break;
-			case MotionEvent.ACTION_CANCEL:
-			case MotionEvent.ACTION_UP: {
-				GodotLib.touch(2,0,evcount,arr);
-				//for(int i=0;i<event.getPointerCount();i++) {
-				//	System.out.printf("%d - up! %f,%f\n",i, event.getX(i),event.getY(i));
-				//}
-			} break;
+		switch (event.getAction() & MotionEvent.ACTION_MASK) {
+
+		case MotionEvent.ACTION_DOWN: {
+			GodotLib.touch(0, 0, evcount, arr);
+			//System.out.printf("action down at: %f,%f\n", event.getX(),event.getY());
+		}
+			break;
```

Comment:
That `switch` style is a bit weird, and different to the one used in our C++ code, even though syntactically it's the same. Can you configure the linter to have a similar output as `clang-format` 5.0 does with our current config for C++?

Reply:
Hum I will use clang-formatter and clean-up this part of the code also (using `{ ... }` or `break` but not both).

Reply:
I'm not familiar with Java, is the `break` unnecessary if you use `{ ... }`? In C/C++, you need it, otherwise you fall through to the next case: https://stackoverflow.com/questions/252489/why-was-the-switch-statement-designed-to-need-a-break